desktop/cmp/treemap/TreeMapModel.ts

```diff
@@ -257,7 +257,7 @@ export class TreeMapModel extends HoistModel {
     }
 
     @computed
-    get expandState(): Record<string, any> {
+    get expandState(): PlainObject {
```

Comment:
actually, I think this should be of type 'any' for now to match the type of the underlying grid model api.  (for better or for worse)